apps/synapse-oauth-signin/src/AppInitializer.tsx

```diff
@@ -146,14 +140,17 @@ function AppInitializer(
   }, [])
 
   return (
-    <OAuthAppContext.Provider value={{ accessToken, setAccessToken }}>
+    <OAuthAppContext.Provider
+      value={{ accessToken, setAccessToken, twoFactorAuthErrorResponse }}
+    >
       <SynapseContextProvider
         synapseContext={{
           accessToken: accessToken,
           isInExperimentalMode: SynapseClient.isInSynapseExperimentalMode(),
           utcTime: SynapseClient.getUseUtcTimeFromCookie(),
           downloadCartPageUrl: '',
         }}
+        theme={themeOptions}
```

Comment:
pass the app's theme into SynapseContextProvider. Maybe later it would make sense for SynapseContextProvider to check if there's a present theme in context instead of accepting it as a prop